src/main/java/Domain/Exceptions/ShippingFailedException.java

```diff
@@ -0,0 +1,12 @@
+package Domain.Exceptions;
+
+public class ShippingFailedException extends RuntimeException{
```

Comment:
// This class represents an error when ...

Reply:
Done.